tests/conftest.py

```diff
@@ -752,6 +752,25 @@ def stream_dict(streams_fixture):
     return {stream['stream_id']: stream for stream in streams_fixture}
 
 
+@pytest.fixture(params=[
+        {
+            ('Stream 2', 'muted topic'): 1530129122,
+            ('Stream 1', 'muted stream muted topic'): 1530129122,
+        },
+        {
+            ('Stream 2', 'muted topic'): None,
+            ('Stream 1', 'muted stream muted topic'): None,
+        },
+    ],
+    ids=[
+        'server_feature_level:1',
+        'server_feature_level:None',
+    ]
+)
+def muted_topics_dict(request):
```

Comment:
This is the parametrization of muted_topics in the Model only - we should likely have one for initial data from server, and a test for initial data -> model data, which we don't have? (that was in the previous iteration?)

Reply:
I didn't have that test in the previous iteration. Thanks for pointing that out.

I have added the test in the latest update with a local `parametrize` block in `Model.py`. It wouldn't be needed in `conftest` as we'd only test against `processed_muted_topics` everywhere else.

Reply:
I'm actually thinking we might move the server response fixtures into a separate fixture file specific to the model in a similar way. What do you think?

Reply:
The ones that are already in `conftest.py` or also the ones that are used in `test_model.py`?

Reply:
In any case, I believe it is reasonable.